docs/001_develop/03_client-capabilities/004_header/02_customise.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Customise Header'
+sidebar_label: 'Customise'
```

Comment:
nitpick, should side bar label be ' Customise header`

Header should be lower case in both cases